sdk/storage/ci.functions.yml

```diff
@@ -0,0 +1,37 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger: none
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/storage/ci.webjobs.yml
+    - sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage/
+    - sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/
+    - sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/
+    - sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/
+    - sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: storage
```

Comment:
You will need to set SDKType in each of these yml files as well. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/ci.functions.yml#L31

Reply:
Done

Reply:
Thanks, Wes.   I overlooked that!